tests/testPupilCoordinates.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import division
+from builtins import zip
+from builtins import range
+from past.utils import old_div
```

Comment:
Before you spend too much time on this, Tim, I think you should just look at the sum of all changes (i.e. there is a commit after the one you are reviewing which takes out all of the occurrences of old_div)